README.md

```diff
@@ -20,6 +31,8 @@ The [NAIPs come from a requester pays bucket on S3 set up by Mapbox](http://www.
 
 ## Install Requirements
 
+DeepOSM has been run successfully on both Mac and Linux (14.04 and 16.04). You need at least 4GB of memory.
```

Comment:
With the current defaults, it isn't necessary. This is probably even conservative now.

@silberman can run it on a Mac with 4GB and I think only allocate 2GB to VirtualBox.